docs/source/dataframe-sql.rst

```diff
@@ -0,0 +1,170 @@
+Dask Dataframe and SQL
+======================
+
+SQL is a method for executing tabular computation on database servers.
+Simlar operations can be done on Dask Dataframes. Users commonly wish
+to link the two:
+
+- perform SQL-like operations on Dask Dataframes
+
+- load data from databases using SQL, and process the results in Dask
+
+- save the output of Dask commputations in a database
+
+- use the Dask API to specify queries, but execute some or part of the
+  query on the database
+
+- which is the best tool for my workload, SQL or dask?
```

Comment:
I don't think this belongs in the list of what people like to do

```suggestion

```

Reply:
I'll probably replace the list with an in-page TOC eventually